Assets/MixedRealityToolkit/Inspectors/ServiceInspectors/ServiceFacadeInspector.cs

```diff
@@ -0,0 +1,405 @@
+                buttonContent.image = EditorGUIUtility.IconContent("_Help").image;
+                buttonContent.text = " Documentation";
+                buttonContent.tooltip = docLink.URL;
+
+                GUILayout.BeginHorizontal();
+                GUILayout.FlexibleSpace();
+
+                if (GUILayout.Button(buttonContent, GUILayout.MaxWidth(docLinkWidth)))
+                {
+                    Application.OpenURL(docLink.URL);
+                }
+
+                GUILayout.FlexibleSpace();
+                GUILayout.EndHorizontal();
+                EditorGUILayout.Space();
+                return true;
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// Draws a list of services that use this as a data provider
+        /// </summary>
+        /// <param name="serviceType"></param>
+        /// <returns></returns>
+        private bool DrawDataProviders(Type serviceType)
+        {  
+            // If this is a data provider being used by other services, mention that now
+            dataProviderList.Clear();
+            foreach (MixedRealityDataProviderAttribute dataProviderAttribute in serviceType.GetCustomAttributes(typeof(MixedRealityDataProviderAttribute), true))
+            {
+                dataProviderList.Add(" • " + dataProviderAttribute.ServiceInterfaceType.Name);
+            }
+
+            if (dataProviderList.Count > 0)
+            {
+                EditorGUILayout.HelpBox("This data provider is used by the following services:\n " + String.Join("\n", dataProviderList.ToArray()), MessageType.Info);
+                EditorGUILayout.Space();
+                return true;
+            }
+            return false;
+        }
+               
+        /// <summary>
+        /// Draws the custom inspector gui for service type.
+        /// </summary>
+        /// <param name="facade"></param>
+        /// <returns></returns>
+        private bool DrawInspector(ServiceFacade facade)
+        {
+            IMixedRealityServiceInspector inspectorInstance;
+            if (GetServiceInspectorInstance(facade.ServiceType, out inspectorInstance))
+            {
+                inspectorInstance.DrawInspectorGUI(facade.Service);
+                return true;
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// Draws the profile for the service type, if wanted by inspector and found.
+        /// </summary>
+        /// <param name="serviceType"></param>
+        /// <returns></returns>
+        private bool DrawProfile(Type serviceType)
+        {
+
+            IMixedRealityServiceInspector inspectorInstance;
+            if (GetServiceInspectorInstance(serviceType, out inspectorInstance))
+            {
+                if (!inspectorInstance.DrawProfileField)
+                {   // We've been instructed to skip drawing a profile by the inspector
+                    return false;
+                }
+            }
```

Comment:
Super Nit: 
if (GetServiceInspectorInstance(serviceType, out inspectorInstance) && !inspectorInstance.DrawProfileField) 
{
  return false;
}

It's just adding an unnecessary extra scope.